src/plugins/users/user-op.js

```diff
@@ -66,12 +67,13 @@ export class UserOp {
       if (!util.emptyString(blocklistFlag) && util.emptyObj(r)) {
         r = rdnsutil.unstamp(blocklistFlag);
 
-        // FIXME: add to cache iff !empty(r.userBlocklistFlagUint)?
-        this.log.d(ctx.rxid, "new cfg cache kv", blocklistFlag, r);
-        // TODO: blocklistFlag is not normalized, ie b32 used for dot isn't
-        // converted to its b64 form (which doh and rethinkdns modules use)
-        // example, b32: 1-AABABAA / equivalent b64: 1:AAIAgA==
-        this.userConfigCache.put(blocklistFlag, r);
+        if (!bufutil.emptyBuf(r.userBlocklistFlagUint)) {
+          this.log.d(ctx.rxid, "new cfg cache kv", blocklistFlag, r);
+          // TODO: blocklistFlag is not normalized, ie b32 used for dot isn't
+          // converted to its b64 form (which doh and rethinkdns modules use)
+          // example, b32: 1-AABABAA / equivalent b64: 1:AAIAgA==
+          this.userConfigCache.put(r.userBlocklistFlagUint, r);
```

Comment:
make this correction pls: The cache key is `blocklistFlag` (which is a string that looks like `1:8AcHAP__________-f8YAA==`) and not `r.userBlocklistFlagUint`.